.github/workflows/markdown-link-check.yml

```diff
@@ -1,7 +1,9 @@
 name: Check Markdown links
 
 on:
-  pull_request:
```

Comment:
why change from PR to push - glad you asked! 

If you add new docs and new link, your PR will fail - see https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/407

The link won't be live and available to test properly until after merging and the GH pages are live. We also might want to bake this into the gh-page deploy pipeline and add a badge or something to ensure links are all good on `main`